### DISCUSSION TITLE
Student developer pack -ISSUE

### DISCUSSION THREAD

**Discussion by floo69** [community/community#127960] [GitHub Education]
### Hi everyone,

i had signed up for student developer pack 1 week ago and when i use to check the website i used to see this image.
![student-developer-pack-doubt-v0-v7qrv5zmax3d1](https://github.com/community/community/assets/139336360/82fc8d70-e244-4b70-b531-1f0afb82b103)
But now when i open the website i am not getting any message or anything. How can i know ?
![image](https://github.com/community/community/assets/139336360/7a18cc80-fd9c-4cc7-b7a8-92118044c0b1)
Its empty without any message

**Comment by SjxSubham** [comment]
Hey @floo69 , click and try to apply into GitHub education again, it will redirect you to main page where it will show you the status... If is it pending to be get verified or your proposal got rejected.
If rejected don't worry try to apply again and this time give your College/School ID as a valid proof ID

But first check Ur status first, Once  u get the GitHub Developer pack, **PRO** Batch will be showed in your profile 
I hope this will help to get Ur answer **:-)**

**Reply by floo69** [reply]
thats the problem, it is not showing me the status, and it is not asking for student mail id , it just asks college mail and i gave it and it got verified but cannot press the continue button. 
![image](https://github.com/community/community/assets/139336360/ce99252c-88da-49ad-bedd-e9cf33fcc4f8)

**Reply by CodeModeYT** [reply]
if you can't press the button, this might be because the email adress you entered is not a school-issued mail adresses (at least in githubs opinion in this case).

Double check that this is the email adress that got issued by your school and that you followed all the steps written in the screenshot you applied. If everything seems ok to you, don't hesitate contacting github support (they are incredibly friendly and helpful) and / or your schools IT department.
 
Hope this helps!

**Reply by SjxSubham** [reply]
well, set a primary email address in ur github account settings...
then activate 2 factor authentication via mobile
then try again... if sometime occurs if u have not set a primary email or enabled 2factor authorised ...
then apply for the same, hope it works...

**Reply by CodeModeYT** [reply]
I just noticed that the page you showed also shows up for me, I already have the GitHub education pack.

And: never hesitate to re-apply, took me 6 tries too:
![grafik](https://github.com/community/community/assets/99124573/ca6a7789-d6df-4984-84f3-d2c22e8967cd)

**Reply by floo69** [reply]
Thanks@CodeModeYT
 , i will try but the issue is i don't even know whether it is processing stage or failed.
The college mail is proper but i will wait and then contact support

**Reply by floo69** [reply]
@SjxSubham Thanks, i will try this 2 factor.

**Reply by SjxSubham** [reply]
@floo69 If u tried those steps and got ur answer u can close this discussion as resolved... :-)
